### DISCUSSION TITLE
Insert Line Break In the Title

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#7564] [Help]
Hi All,
I'm using filament 3
I want to insert line break in the title, but \n not working.
![image](https://github.com/filamentphp/filament/assets/6214964/b2b06418-2f33-4cfe-bae9-c6a61cc23484)
How to make line break in the title ?

And I just need to show 'Mega Mall' in dashboard page
![image](https://github.com/filamentphp/filament/assets/6214964/55b7f01a-8d37-472d-b415-c69a1e099baf)

thanks

**Comment by leandrocfe** [comment]
create the file resources/views/vendor/filament-panels/components/logo.blade.php

```html
<div class="flex flex-col items-center justify-center">
    <span class="font-semibold">Latest Shop List</span>
    <span class="font-semibold">Mega Mall</span>
</div>
```
https://filamentphp.com/docs/3.x/panels/themes#changing-the-logo

**Comment by jigujigu** [comment]
this is my code in logo.blade.php

```
<div class="flex flex-col items-center justify-center">
    <span class="text-3xl font-bold">Latest Shop List</span>
    <span class="font-semibold">Mega Mall</span>
</div>
```

![image](https://github.com/filamentphp/filament/assets/6214964/29af4f70-2d6d-4bca-a66c-2b87628cac1a)


But I want only become 'Mega Mall' on the top of left dashboard like this 
![image](https://github.com/filamentphp/filament/assets/6214964/0f091f75-59ba-4185-9a5a-fcfe14a9d39c)

**Reply by leandrocfe** [reply]
What is the issue? I didn't understand
